docs/resources/wafv2_logging_configuration.md

```diff
@@ -13,16 +13,21 @@ A WAFv2 Logging Configuration Resource Provider
 
 ### Basic usage
 
-Creates a WAFv2 Web ACL Logging Configuration resource. Note that AWS Provider resource for [aws_wafv2_web_acl](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl) is used.
+Creates a WAFv2 Web ACL Logging Configuration resource. Note that AWS Provider resources for [aws_wafv2_web_acl](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl) and [aws_cloudwatch_log_group] (https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group) are used.
 
 ```terraform
 resource "awscc_wafv2_logging_configuration" "awscc_waf_logging" {
   resource_arn            = aws_wafv2_web_acl.example.arn
-  log_destination_configs = [awscc_logs_log_group.example.arn]
+  log_destination_configs = [aws_cloudwatch_log_group.example.arn]
+
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
 }
 
-resource "awscc_logs_log_group" "example" {
-  log_group_name = "aws-waf-logs-awscc"
+resource "aws_cloudwatch_log_group" "example" {
```

Comment:
Hi @jackywong-amazon, I noticed you changed  `awscc_logs_log_group` to `aws_cloudwatch_log_group`. If the resource exists in the AWSCC provider should we use it instead of the AWS provider?

Reply:
there is a bug for awscc_logs_log_group and it would cause the wafv2 log unable to push to the created cloudwatch log group (AWSCC). more info please check out (https://github.com/hashicorp/terraform-provider-awscc/issues/1028).

I have tested both providers and the final outcome is that, AWSCC one would cause the end user issue when looking within the WAFv2 console (when click on the log group in console, it would redirect to the error page) while AWS one would not cause any issue. I would strongly to keep AWS one until the AWSCC CW log group bug is fixed.

Reply:
Thanks for the detailed testing and explanation @jackywong-amazon! PR LGTM!